product_docs/docs/biganimal/release/getting_started/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Understanding requirements in Azure"
+
+---
+
+By default, Azure sets low resource limits on the number of vCPUs (cores), and Public IP addresses permitted in your subscription. The following sections explain BigAnimal's requirements and recommended resource limits in Azure.
+
+### Public IP addresses limits
+
+A Public IP address is an IP address that used to access your cluster directly over the internet. The number of Public IP addresses must not exceed the limit for a particular region.
+
+##### Default Azure limit
+
+The default Public IP addresses limits for **Basic** and **Standard** type are set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information.
+
+##### Recommended limit
+
+EDB recommends you to increase the Public IP addresses limits for **Basic** and **Standard** type to the number of clusters you plan to deploy in a given region.
+
+### vCPU limits
+
+Any time a new VM is deployed in Azure, the vCPUs for the VMs must not exceed the vCPU limits for the VM size family (see [SKU restrictions](#SKU-restrictions)) or the total regional vCPU limit. Clusters deployed in the region use separate Esv3 virtual machine cores. 
```

Comment:
SKU restrictions and VM quotas are different. And does this link go anywhere?

Reply:
@ba-edb see https://github.com/EnterpriseDB/docs/pull/1962#discussion_r762071810 and let me know if this issue is resolved?